src/lib.rs

```diff
@@ -1415,6 +1428,12 @@ pub trait WebViewBuilderExtDarwin {
   /// Warning: Do not use this if your chosen window library does not support traffic light insets.
   /// Warning: Only use this in **decorated** windows with a **hidden titlebar**!
   fn with_traffic_light_inset<P: Into<dpi::Position>>(self, position: P) -> Self;
+  /// Whether to show a link preview when long pressing on links. Available on macOS and iOS only.
+  ///
+  /// Default is true.
+  ///
+  /// See https://docs.rs/objc2-web-kit/latest/objc2_web_kit/struct.WKWebView.html#method.allowsLinkPreview
+  fn allow_link_preview(self, allow_link_preview: bool) -> Self;
```

Comment:
Can we use `with_link_preview` to align with other builder method's naming style